pages/api/graphql-rest.page.ts

```diff
@@ -11,17 +10,10 @@ import {
   ExportLabelTypeEnum,
   ExportSortEnum,
 } from '../../graphql/types.generated';
-import {
-  ContactFilterOption,
-  ContactFilterGroup,
-  ContactFilter,
-} from './graphql-rest.page.generated';
-import ContactFiltersResolvers from './Schema/ContactFilters/resolvers';
-import TaskAnalyticsResolvers from './Schema/TaskAnalytics/resolvers';
-import ExportContactsResolvers from './Schema/ExportContacts/resolvers';
-import ContactFiltersTypeDefs from './Schema/ContactFilters/typeDefs';
-import TaskAnalyticsTypeDefs from './Schema/TaskAnalytics/typeDefs';
-import ExportContactsTypeDefs from './Schema/ExportContacts/typeDefs';
+import { ContactFilterOption } from './graphql-rest.page.generated';
+import schema from './Schema';
+import getTaskAnalytics from './Schema/TaskAnalytics/dataHandler';
+import getContactFilters from './Schema/ContactFilters/datahandler';
 
 class MpdxRestApi extends RESTDataSource {
```

Comment:
Also thinking of moving this out and into its own file. Let me know what you guys think.